firebase-config/test-app/src/main/kotlin/com/google/firebase/testing/config/MainActivity.kt

```diff
@@ -17,14 +17,28 @@
 package com.google.firebase.testing.config
 
 import android.os.Bundle
+import android.util.Log
+import android.widget.Button
 import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
 
 class MainActivity : AppCompatActivity() {
+
+  private lateinit var remoteConfig: FirebaseRemoteConfig
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_main)
 
     findViewById<TextView>(R.id.greeting_text).text = getText(R.string.firebase_greetings)
+    findViewById<Button>(R.id.crash_button).setOnClickListener {
```

Comment:
Can we call this "JVM Crash"? Now I am thinking about adding more type of crashes and integrate with NDK later for this test app later.

Reply:
SGTM. Updated.